pkg/constants.go

```diff
@@ -14,4 +14,5 @@ const (
 	PathOpenAPI              = "/.well-known/openapi.json"
 	PathSwagger              = "/.well-known/swagger"
 	PathSwaggerWithPathParam = "/.well-known/swagger/{name}"
+	FrameworkMetricsPrefix   = "zs_"
```

Comment:
Should we change all the metrics prefix to gofr instead of zs?

Reply:
Yes, we can do that.

Reply:
Tracking the issue here: https://github.com/gofr-dev/gofr/issues/99